webpack.config.js

```diff
@@ -11,18 +11,17 @@ function resolve (dir) {
 }
 
 const config = {
-  entry: [
-    resolve('./src/index.js')
-  ],
+  entry: resolve('./src/index.js'),
   output: {
     path: path.resolve(__dirname, "dist"),
     filename: 'demosplan-ui.umd.js',
-    library: {
-      type: 'umd'
-    },
+    library: '@demos-europe/demosplan-ui',
+    libraryTarget: 'umd',
+    libraryExport: "default"
```

Comment:
Nitpick: single quotes

Reply:
adjusted in 5aae9baf51ec19db59531588614bd3f8c7aee519